Modules/StudyProgramme/classes/class.ilObjStudyProgrammeMembersGUI.php

```diff
@@ -950,18 +950,16 @@ public function mayManageMembers() : bool
         );
     }
 
-    public function getLocalMembers(): array
+    public function getLocalMembers() : array
     {
         return $this->object->getMembers();
     }
 
-    public function isOperationAllowedForUser(int $usr_id, string $operation): bool
+    public function isOperationAllowedForUser(int $usr_id, string $operation) : bool
     {
-        $ret = $this->position_based_access->isUserAccessibleForOperationAtPrg(
-            $usr_id,
-            $this->object,
-            $operation
-        );
+        $ret = $this->position_based_access->isUserAccessibleForOperationAtPrg($usr_id, $this->object, $operation)
```

Comment:
Does `isUserAccessibleForOperation` check if the the position system is activated for the Study Programme already? If not, we would lose this check, right?

Reply:
I think it  does; 
ilOrgUnitPositionAccess::filterUserIdsByPositionOfCurrentUser

        // If context is not activated, return same array of $user_ids
        if (!$this->set->getObjectPositionSettingsByType($this->getTypeForRefId($ref_id))->isActive()) {
            return $user_ids;
        }